tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -250,16 +251,25 @@ func errorf(t *testing.T, format string, args ...interface{}) {
 	t.Errorf(format, args...)
 }
 
-func TestGlobalCheckAndReport(t *testing.T) {
+func TestGlobalReport(t *testing.T) {
 	// setup prometheus API
 	promAPI, err := promAPI()
 	if err != nil {
 		t.Fatalf("Could not build prometheus API client: %v", err)
 	}
 
-	// establish baseline
-	t.Log("Establishing metrics baseline for test...")
-	query := fmt.Sprintf("istio_request_count{%s=\"%s\"}", destLabel, fqdn("productpage"))
+	checkMetricReport(t, promAPI, "productpage")
```

Comment:
Similar to check Please add 2 top level tests for `TestMetric` and `TestIngressMetric`

Reply:
Done.